opennms-doc/guide-install/src/asciidoc/text/opennms/rhel.adoc

```diff
@@ -57,6 +57,24 @@ With the successful installed packages the _OpenNMS_ platform is installed in th
    └── system
 ----
 
+[[gi-install-opennms-rhel-disable-yum-repo]]
+==== Disable Automatic Updates (Optional)
+
+We recommend you disable the OpenNMS Yum repository to avoid upgrades while OpenNMS is running.
+
+OpenNMS requires some manual steps upon upgrade to make sure the database and configuration are consistent in the new version, and on systems with many nodes or lots of events, this can take hours.  For this reason, it is recommended to exclude the OpenNMS packages from yum except when you are planning on performing an upgrade.
+
+To do so, edit the `/etc/yum.repos.d/opennms-*.repo` file and change `enabled=1` to `enabled=0` in each section.
+
+When you are ready to upgrade OpenNMS, call yum with the `--enablerepo` option to turn the 2 repositories defined in this file back on.  For example, if you installed the stable repository RPM on a CentOS or RHEL 7 system, you would run:
```

Comment:
We should use `_{opennms-product-name}_` instead of OpenNMS.
New line for each sentence to make diff fixes easier.
Formatting: `_RPM_`, `_CentOS_`, `_RHEL 7_`.